support/client/lib/vwf/view/mil-sym.js

```diff
@@ -368,25 +369,39 @@ define( [ "module", "vwf/view", "mil-sym/cws", "jquery" ], function( module, vie
 
         var rendererMP = sec.web.renderer.SECWebRenderer;
         var scale = 100.0;
-        var updatedUnit = $.extend( true, {}, unit );
         var appID = self.kernel.application();
         var renderer = armyc2.c2sd.renderer;
         var msa = renderer.utilities.MilStdAttributes;
         var rs = renderer.utilities.RendererSettings;
         var symUtil = renderer.utilities.SymbolUtilities;
+        var milSymControlPts = convertKonvaToMilSymControlPts( controlPoints );
         
         // Set affiliation in symbol id
         symbolCode = cws.addAffiliationToSymbolId( symbolID, affiliation );
         
-        var img = rendererMP.RenderSymbol2D("ID","Name","Description", symbolCode, controlPoints, bounds.width, bounds.height, null, modifiers, format);
+        var img = rendererMP.RenderSymbol2D("ID","Name","Description", symbolCode, milSymControlPts, bounds[0], bounds[1], null, modifiers, format);
 
-        if ( !!img && !!img.image ) {  
-            return img.image;
-        }      
+        if ( !!img ) {
+            return img;
+        }    
```

Comment:
Since, if `img` is falsy, this function will return `undefined`, can you just `return img;` without the condition?